microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/advanced/JWTSecurityAnnotationsTest.java

```diff
@@ -0,0 +1,184 @@
+    }
+
+    static class OpenApiExtensionSetup implements ServerSetupTask {
+
+        @Override
+        public void setup(ManagementClient managementClient, String containerId) throws Exception {
+            //  MP OpenAPI up
+            onlineManagementClient = ManagementClientProvider.onlineStandalone();
+            OpenApiServerConfiguration.enableOpenApi(onlineManagementClient);
+        }
+
+        @Override
+        public void tearDown(ManagementClient managementClient, String containerId) throws Exception {
+            //  MP OpenAPI down
+            try {
+                OpenApiServerConfiguration.disableOpenApi(onlineManagementClient);
+            } finally {
+                onlineManagementClient.close();
+            }
+        }
+    }
+
+    /**
+     * @tpTestDetails Test to verify that the app secured endpoints are up and running
+     * @tpPassCrit One of the Local Service Router endpoints is called to retrieve data from Service Provider,
+     *             verifying the HTTP response code and content type
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void testAppEndpoint(@ArquillianResource @OperateOnDeployment(ROUTER_DEPLOYMENT_NAME) URL baseURL) {
+        get(baseURL.toExternalForm() + "info/fqdn")
+                .then()
+                .statusCode(200)
+                //  RestEasy >3.9.3 adds charset info unless resteasy.add.charset is set to false,
+                //  thus expecting for it to be there, see
+                //  https://docs.jboss.org/resteasy/docs/3.9.3.Final/userguide/html/Installation_Configuration.html#configuration_switches
+                .contentType(equalToIgnoringCase("text/plain;charset=UTF-8"));
+    }
+
+    /**
+     * @tpTestDetails Test to verify that information derived from {@code @SecuritySchemes} annotations - in this case
+     *                related to JWT - is contained in generated OpenAPI document, see
+     *                https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/quarkus-dev/QZgqrblsYEc/sgDcnHYlBAAJ
+     * @tpPassCrit OpenAPI endpoint is called to verify its availability (HTTP response code 200) and then
+     *             the returned data is checked to asses that operations in the generated OpenAPI document do contain
+     *             information
+     *             about the JWT related {@code @SecurityScheme} which is placed just over the JAX-RS resource class
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testForSecurityAnnotationsForAllOperations(
+            @ArquillianResource @OperateOnDeployment(ROUTER_DEPLOYMENT_NAME) URL baseURL) throws URISyntaxException {
+        String responseContent = get(baseURL.toURI().resolve("/openapi"))
+                .then()
+                .statusCode(200)
+                .extract().asString();
+
+        Yaml yaml = new Yaml();
+        Object yamlObject = yaml.load(responseContent);
```

Comment:
Can you directly cast the Object to `Map<String, Object>` and create `yamlMap` here?

Reply:
Sure, you're right. Actually `load` is returning `Map` already, so no need to cast.

Reply:
@fabiobrz I can see the same thing in other test classes for example `MicroProfileOpenApi11Test`, do you think you might change it there as well, please?